README.md

```diff
@@ -88,14 +88,13 @@ More process details to follow, it's currently being worked out.
 
 <summary>Bastion configuration</summary>
 
-Update the user line to your email address
+Ensure you have Azure CLI version 2.22.1 or later installed
```

Comment:
this line needs removing 'Add SSH config, this goes in `~/.ssh/config`, create the file if it doesn't exist'